src/pages/workflow-execution/WorkflowExecution.tsx

```diff
@@ -114,20 +98,35 @@ export const WorkflowExecution = () => {
     getTaskWorkflowExecutionData(guid);
   };
 
-  useEffect(
-    () => () => {
-      clearBusy();
-    },
-    [],
-  );
+  const updateDurationTimes = () => {
+    if (workflowExecutionData && workflowExecutionData.length > 0) {
+      setDurationTimes(
+        workflowExecutionData.map((we) => ({
+          guid: we.guid,
+          duration: calculateDuration(we),
+        })),
+      );
+    }
+  };
+
+  useEffect(() => {
+    const durationInterval = setInterval(() => {
+      setTick((prev) => !prev);
+    }, 1000);
+
+    return () => clearInterval(durationInterval);
+  }, []);
+
+  useEffect(() => {
+    updateDurationTimes();
+  }, [tick]);
 
   const backupDestinationsBody = (backupDestinations) =>
     backupDestinations.map(({ name }) => name).join('-');
 
   const refresh = () => {
     setPage(0);
     setWorkflowExecutionData([]);
-    expandedRows?.forEach(({ guid }) => getTaskWorkflowExecutionData(guid));
```

Comment:
this has been moved to the line 80 (so now it will update straight from API data)